Packs/FeedMitreAttack/Integrations/FeedMitreAttack/FeedMitreAttack.py

```diff
@@ -1,12 +1,11 @@
 import demistomock as demisto
 from CommonServerPython import *
-from CommonServerUserPython import *
 
 from typing import List, Dict, Set
 import json
 import requests
 from stix2 import TAXIICollectionSource, Filter
-from taxii2client import Server, Collection, ApiRoot
+from taxii2client.v20 import Server, Collection, ApiRoot
```

Comment:
make sure you're synced with https://github.com/demisto/etc/issues/26353 with these change

Reply:
Synced, He implemented the same solution